unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Messaging/MessagingControllersManager.cs

```diff
@@ -17,10 +17,12 @@ public class MessagingControllersManager : IMessagingControllersManager
         private const float GLTF_BUDGET_MAX = 0.033f;
         private const float GLTF_BUDGET_MIN = 0.008f;
 
-        public const string GLOBAL_MESSAGING_CONTROLLER = "global_messaging_controller";
+        // public const string GLOBAL_MESSAGING_CONTROLLER = "global_messaging_controller";
+        public const int GLOBAL_MESSAGING_CONTROLLER_SCENE_NUMBER = -100; // TODO: Does this bring trouble with < 0 checks?
```

Comment:
you do have many places where you check that the validity of the scene number must be > 0

Reply:
uhmm yes this is old, from the beginning of the implementation, let me re-check that, thanks!

Reply:
Yeap, that had no sense, even some parts of the that class initialization [didn't make any sense](https://github.com/decentraland/unity-renderer/blob/dev/unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Messaging/MessagingControllersManager.cs#L61~L65) in `dev` (storing the messaging controller in the dictionary and immediately checking if the value existed, and also checking if a const string is null/empty WTF), so I fixed it [here](https://github.com/decentraland/unity-renderer/pull/2977/commits/233db652647c8394a98a62436a97bdee694acd97).
Thanks!